src/main/webapp/i18n/zh-cn/global.json

```diff
@@ -125,7 +126,8 @@
       "view": "查看",
```

Comment:
line 124 open is not translated

Reply:
Resolved in 2d6dc9ee70121543ae0be8433fcfc8f9801146cc